src/pages/Main/index.js

```diff
@@ -1,42 +1,91 @@
 import React from 'react';
 import Form from '../../components/Form';
-import List from '../../components/List';
+import ListAlta from '../../components/ListAlta';
+import ListMedia from '../../components/ListMedia';
+import ListBaixa from '../../components/ListBaixa';
 
 class Main extends React.Component {
   constructor(props){
     super(props);
 
     this.state = {
-      data: [],
+      data: [{"Alta": [], "Media": [], "Baixa": []}],
     }
   }
 
-  addTodo(event, val){
+  addTodo(event, val, prioridade){
     event.preventDefault();
+    
+    var todo = { text: val };
+    if (prioridade === "") {
```

Comment:
Ao invés de setar aqui que se a prioridade for vazia, ela deve ser alta. A melhor prática seria colocar isso no estado inicial